contracts/schemes/ContinuousLockingToken4Reputation.sol

```diff
@@ -41,17 +41,17 @@ contract ContinuousLocking4Reputation is Agreement {
     mapping(uint256 => Batch) public batches;
 
     Avatar public avatar;
-    uint256 public reputationRewardLeft;
-    uint256 public startTime;
+    uint256 public reputationRewardLeft; // the amount of reputation  that is still left to disctribute
+    uint256 public startTime;  // the time (in secs since epoch) that locking can start
     uint256 public redeemEnableTime;
     uint256 public maxLockingBatches;
-    uint256 public batchTime;
-    IERC20 public token;
-    uint256 public batchesCounter; // Total number of batches
-    uint256 public totalLockedLeft;
-    uint256 public repRewardConstA;
-    uint256 public repRewardConstB;
-    uint256 public batchesIndexCap;
+    uint256 public batchTime; // the length of a batch, in seconds
+    IERC20 public token; // the token to be locked
+    uint256 public batchesCounter; // [The batchesCounter is incremented on each lock - more appropriate to name it "lockCounter"?]
+    uint256 public totalLockedLeft; //  the total amount of tokens locked in the contract [JG: not sure why we need this - can't we just use the balance of this contrcat?]
```

Comment:
it is used as additional "layer" to make sure there is no rep allocation other than needed. 
here
https://github.com/daostack/arc/blob/35b387ae36b2ea979c3b291fc044d8b340b31434/contracts/schemes/ContinuousLockingToken4Reputation.sol#L264 .